v3/util/etcdctl-auth-root.sh

```diff
@@ -0,0 +1,71 @@
+
+#!/bin/bash
+
+source /etc/environment
+
+#Disable authentication
+
+#curl  -L http://127.0.0.1:2379/v2/auth/enable -XDELETE
+
+etcdctlrootusername=$(etcdctl get /etcdctl/config/root-user)
+etcdctlrootpassword=$(etcdctl get /etcdctl/config/root-password)
+etcdctletcdreadusername=$(etcdctl get /etcdctl/config/read-user)
+etcdctletcdreadpassword=$(etcdctl get /etcdctl/config/read-password)
+etcdctletcdreadwriteusername=$(etcdctl get /etcdctl/config/write-user)
+etcdctletcdreadwritepassword=$(etcdctl get /etcdctl/config/write-password)
+
+#add a root user
+
+echo '{"user":"'$(eval echo $etcdctlrootusername)'", "password":"'$(eval echo $etcdctlrootpassword)'"}' > /home/core/mesos-systemd/v3/util/root.json
+curl  -L http://127.0.0.1:2379/v2/auth/users/$(eval echo $etcdctlrootusername) -XPUT -d "@/home/core/mesos-systemd/v3/util/root.json"
+
+
+#add a non root read only user
+echo '{"user":"'$(eval echo $etcdctletcdreadusername)'", "password":"'$(eval echo $etcdctletcdreadpassword)'"}' > /home/core/mesos-systemd/v3/util/etcdreaduser.json
+curl   -L http://127.0.0.1:2379/v2/auth/users/$(eval echo $etcdctletcdreadusername) -XPUT -d "@/home/core/mesos-systemd/v3/util/etcdreaduser.json"
+
+#add a non root readwrite user
+echo '{"user":"'$(eval echo $etcdctletcdreadwriteusername)'", "password":"'$(eval echo $etcdctletcdreadwritepassword)'"}' > /home/core/mesos-systemd/v3/util/etcdwriteuser.json
```

Comment:
Change `/home/core/mesos-systemd/v3/util/etcdwriteuser.json` to `/opt/etcdctl/write-user.json`